src/Http/Place/ImportPlaceRequestHandler.php

```diff
@@ -283,4 +263,26 @@ public function handle(ServerRequestInterface $request): ResponseInterface
         ];
         return new JsonResponse($responseBody, $responseStatus);
     }
+
+    public function guardDuplicatePlace(Place $place): void
+    {
+        if (! $this->preventDuplicatePlaces) {
+            return;
+        }
+
+        try {
+            $duplicatePlaceId = $this->lookupDuplicatePlace->getDuplicatePlaceUri($place);
+            if ($duplicatePlaceId !== null) {
+                throw ApiProblem::statusConflict(
+                    MultipleDuplicatePlacesFound::ERROR_MSG,
```

Comment:
It feels weird to get the error message from the `MultipleDuplicatePlacesFound` although there is exactly one place returned as duplicate.